sdk/template/azure-template2/README.md

```diff
@@ -0,0 +1,70 @@
+# Azure Template Package client library for C++
+
+Azure Template Package client library for C++ (`azure-template`)  matches necessary patterns that the development team has established to create a unified sdk written in the C++ programming language. These libraries follow the Azure SDK Design Guidelines for C++.
```

Comment:
```suggestion
# Azure Template Package 2 client library for C++

Azure Template Package 2 client library for C++ (`azure-template2`)  matches necessary patterns that the development team has established to create a unified sdk written in the C++ programming language. These libraries follow the Azure SDK Design Guidelines for C++.
```

Reply:
Make similar updates elsewhere.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1651